README.md

```diff
@@ -5,33 +5,53 @@ This is a very simple Visual Studio Web Application that includes several text b
 
-#### Browsers Tested:
-Internet Explorer 11, Firefox 38 ESR, Chrome
+#### Supported Aras Versions
 
-> Though built and tested using Aras 11.0 SP9, this project should function in older releases of Aras 11.0.
+| Project                                                      | Aras                                       |
+| ------------------------------------------------------------ | ------------------------------------------ |
+| [v1.0.1](https://github.com/ArasLabs/External-Input-Example/releases/tag/v1.0.1) | 11.0 SP7+, SP12+, SP15; 12.0               |
+| [v1.0.0](https://github.com/ArasLabs/External-Input-Example/releases/tag/v1.0.0) | 11 SP7, SP9; Old Community Board Migration |
+
+## Installation
 
 ### Pre-requisites
 
-1. Aras Innovator installed (version 11.0 SPx preferred)
-2. Visual Studio (2015 preferred)
+1. Aras Innovator installed
+2. Visual Studio (2015+)
+
+### Install Steps
+
+You will need to point the solution at the correct IOM file for your instance of Innovator. It should default to the last version of IOM you have used with Visual Studio, but resetting it to your current version is advised. Steps are as follows:
+
+1. Open the solution in Visual Studio.
+2. In the Solution Explorer pane:
+   1. Under References, remove the IOM package
+   2. Go to the CD Image for your version of Aras and unzip the IOM folder under Utilities to the location of your choice (you will need the .Net folder for this )
+      - Alternate for if you do not have a CD Image: go to the Innovator/Client/bin folder on your install and select the IOM.dll file from there in the below step to add the reference.
+   3. Return the project and add a reference
+   4. Point to the location of the DLL
+3. In the Project Properties (Project pulldown > [project name] Properties…) set the target framework to the version required by your install of Innovator or what the system warnings say regarding the correct version for your IOM file. (You can find this in the system requirements section of the installation documentation.)
+4. Build/rebuild the solution. (And skip to step 2 of usage.)
 
 ## Usage
 
 1. Open the solution in Visual Studio.
 2. Start the External_Input_Example project.
-3. Wait for the the web page to load
+   1. Select the browser you want to run it in.
+3. Wait for the the web page to load.
 4. Enter the log in credentials for the preferred Aras Innovator instance and the new PR data.
-5. Click submit.
+5. Click the submit button.
 6. Once the connection is made and the new PR is created with the data provided by the user, a confirmation window will appear.
 
-![Customized Dashboard](./Screenshots/PR%20Web%20Page%20Filled%20In.PNG)
-*The PR Web Page with log in credentials and PR data filled out*
+![Customized Dashboard](./Screenshots/PR-WebPage2.PNG)
```

Comment:
This url doesn't work. GitHub is case-sensitive, and the actual file name has a lower case file extension.

Reply:
Fixed extension